dts/common/nordic/nrf54l_05_10_15.dtsi

```diff
@@ -371,7 +371,7 @@
 				zephyr,pm-device-runtime-auto;
 			};
 
-			spi22: spi@c8000 {
+			wifi_spi: spi22: spi@c8000 {
```

Comment:
actually wait, what? How does this have any relation to a soc at all ?

Reply:
I just needed a common label between nRF54H20 and nRF54L15 SPIs used for nRF70 device to make the expansion board EB-II shield variants a bit cleanly defined.

Reply:
put it in the board dts, this has no place in soc files